### PR TITLE
feat(trpc): use consistent naming for Trpc exports and use in Nx plugin

### PR DESCRIPTION
**Pull request by goetzrobin**
naming for trpc is inconsistent. this deprecates the inconsistently named variables/functions and introduces variables/functions that all follow the Trpc... naming. While I was at it, I adjusted the nx plugin to use those new names and improved Nx plugin testing. I also moved to a (home).page.ts file and a analog-welcome.component.ts, which will make it easier to delete the generated files.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/analogjs/analog/blob/main/CONTRIBUTING.md#-commit-message-guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Which package are you modifying?

- [ ] vite-plugin-angular
- [ ] vite-plugin-nitro
- [ ] astro-angular
- [ ] create-analog
- [ ] router
- [ ] platform
- [ ] content
- [x] nx-plugin
- [x] trpc

## What is the current behavior?

Inconsistent naming from the trpc package.
Bloated index.page.ts

Closes #

## What is the new behavior?

Deprecated inconsistent naming in favor of Trpc prefixed functions/variables

New analog-welcome.component that encapsulates boilerplate functionality.
Renaming generated page to (home).page.ts

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: apps/nx-plugin-e2e/tests/nx-plugin.spec.ts [hunk 1]
```diff
@@ -1,62 +1,38 @@
...
   it('should create hello-world', async () => {
-    const project = uniq('hello-world');
-    await runNxCommandAsync(`generate @analogjs/platform:app ${project}`);
-    expect(() => checkFilesExist(`apps/${project}/index.html`)).not.toThrow();
-  }, 120000);
+    const project = uniq('app');
+    const res = await runNxCommandAsync(
+      `generate @analogjs/platform:app ${project} --addTailwind=true --addTRPC=true --verbose`
+    );
+    console.log(res.stdout, res.stderr);
```
Comment (brandonroberts) [inline apps/nx-plugin-e2e/tests/nx-plugin.spec.ts:31]: Is this needed?

**Review by brandonroberts — CHANGES_REQUESTED** [review]

File: packages/nx-plugin/src/generators/app/schema.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
   "title": "",
   "type": "object",
   "properties": {
-    "name": {
+    "analogAppName": {
```
[the remark this answers is not in the document]
Comment (goetzrobin) [reply packages/nx-plugin/src/generators/app/schema.json:8]: Currently when using the preset, which uses the app generator under the hood it automatically assigns the workspace name to the application name, too. I think this stems from a conflict between create-nx-workspace and our preset, which both use name. 
The qwik-nx plugin has a similar naming convention:
https://github.com/qwikifiers/qwik-nx/blob/main/packages/qwik-nx/src/generators/preset/schema.json

That makes me 90% certain that this change should allow you to name your nx workspace different to your first analog app.

File: apps/nx-plugin-e2e/project.json [hunk 3]
```diff
@@ -4,16 +4,22 @@
   "projectType": "application",
   "sourceRoot": "apps/nx-plugin-e2e/src",
   "targets": {
+    "build": {
+      "executor": "nx:run-commands",
+      "options": {
+        "command": "nx build platform && nx build trpc && nx build content"
```
[the remark this answers is not in the document]
Comment (goetzrobin) [reply apps/nx-plugin-e2e/project.json]: I didn't get it to work with the nx/plugin:e2e executor. Turns out that one is deprecated already and neither it or the jest executor is even needed. We can use vitest for this too! I changed it to use vitest and fixed some other e2e configs while I was at it 🧹

File: apps/nx-plugin-e2e/tests/nx-plugin.spec.ts (hunk 1 above)
Comment (goetzrobin) [reply apps/nx-plugin-e2e/tests/nx-plugin.spec.ts]: removed! thank you!

**Pull request merged**